languages/python/bitwarden_sdk/bitwarden_client.py

```diff
@@ -43,12 +62,24 @@ def get(self, id: str) -> ResponseForSecretResponse:
         )
         return ResponseForSecretResponse.from_dict(result)
 
-    def create(self, key: str,
-               note: str,
-               organization_id: str,
-               value: str,
-               project_ids: Optional[List[UUID]] = None
-               ) -> ResponseForSecretResponse:
+    def get_by_ids(self, ids: List[UUID]) -> ResponseForSecretsResponse:
+        result = self.client._run_command(
+            Command(secrets=SecretsCommand(
+                get_by_ids=SecretsGetRequest(ids))
+            ))
+        return ResponseForSecretsResponse.from_dict(result)
+
+    def create(
+            self,
+            organization_id: UUID,
+            key: str,
+            value: str,
+            note: str,
```

Comment:
Since `note` can be None, it should be `Optional[str]`

Reply:
Fixed in 86bb6de6d02ac1835cd4e511615b50fe36975fda.